### PR TITLE
fix: FL newest arcgis + deaths

### PR DESCRIPTION
**Pull request by cmcjacob**

### CODE REVIEW

File: src/shared/scrapers/USA/FL/index.js [hunk 1]
```diff
@@ -202,9 +202,38 @@ const scraper = {
...
 
       counties.push(transform.sumData(counties));
-
       counties = geography.addEmptyRegions(counties, this._counties, 'county');
+      return counties;
+    },
+    '2020-3-30': async function() {
+      console.log('USING RIGHT SCRAPER');
+      this.type = 'csv';
+      // this.url = await fetch.getArcGISCSVURL('1', 'a7887f1940b34bf5a02c6f7f27a5cb2c', 'Florida_COVID_Cases');
```
Comment (camjc) [inline src/shared/scrapers/USA/FL/index.js]: Whats the plan with this one? Is there something TODO?

File: src/shared/scrapers/USA/FL/index.js [hunk 2]
```diff
@@ -202,9 +202,38 @@ const scraper = {
...
+        let countyName = geography.addCounty(parse.string(county.County_1));
+        if (countyName === 'Unknown County') {
+          countyName = UNASSIGNED;
+        }
+        if (countyName === 'Dade County') {
+          countyName = 'Miami-Dade County';
+        }
+        if (countyName === 'Desoto County') {
+          countyName = 'DeSoto County';
+        }
```
Comment (camjc) [inline src/shared/scrapers/USA/FL/index.js]: I thought we had a pattern for doing these overrides somewhere? Can someone point to it (or tell me I'm dreaming)?

File: src/shared/scrapers/USA/FL/index.js [hunk 3]
```diff
@@ -202,9 +202,38 @@ const scraper = {
...
 
+        counties.push({
+          county: countyName,
+          cases: parse.number(county.CasesAll || 0),
```
Comment (camjc) [inline src/shared/scrapers/USA/FL/index.js]: Probably we don't want to still use this scraper where cases is falsy, and should throw an error instead?

File: src/shared/scrapers/USA/FL/index.js (hunk 2 above)
Comment (cmcjacob) [reply src/shared/scrapers/USA/FL/index.js]: Yeah I just followed the existing pattern used in this scraper.  Some scrapers use a _countyMap.  Some of them use the rules library to reject certain counties.

File: src/shared/scrapers/USA/FL/index.js (hunk 3 above)
Comment (cmcjacob) [reply src/shared/scrapers/USA/FL/index.js]: Again, referencing the scraper that was there already:
`        counties.push({
          county: countyName,
          cases: parse.number(county.attributes.T_positive || 0),
          tested: parse.number(county.attributes.T_total || 0),
          deaths: parse.number(county.attributes.FLandNonFLDeaths || 0)
        });`

The `|| 0` can safely be removed in this scenario, but I just went off what was already there.

**Comment by cmcjacob** [comment]
@camjc updated PR should address some of these concerns

File: src/shared/scrapers/USA/FL/index.js (hunk 1 above)
Comment (cmcjacob) [reply src/shared/scrapers/USA/FL/index.js]: stray comment removed in 29db10c

**Pull request merged**